gallery/src/components/NxLoadError/NxLoadErrorPage.tsx

```diff
@@ -56,7 +56,17 @@ const NxLoadErrorPage = () =>
             <NxTable.Cell>Function</NxTable.Cell>
             <NxTable.Cell>No</NxTable.Cell>
             <NxTable.Cell>
-              If this is defined, a Retry button will be rendered which executes this function when clicked
+              If this is defined, a Retry button will be rendered with type "button" or "submit" if
+              <NxCode>useSubmitRetry</NxCode> is also set to true,
+              which executes this function when clicked.
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>useSubmitRetry</NxTable.Cell>
```

Comment:
How would you feel about renaming this to `submitRetry`, `submitOnRetry`, or something else?  `use...` typically makes people think of hooks

Reply:
> How would you feel about renaming this to `submitRetry`, `submitOnRetry`, or something else? `use...` typically makes people think of hooks

Thanks yeah I was also trying to think of a better name. Thanks for the suggestions, I went with `submitOnRetry`:
2b1a9ca065d55b324f748a2d776bfe929efc9d53